sdk/textanalytics/ai-text-analytics/src/generated/models/index.ts

```diff
@@ -107,6 +107,24 @@ export interface DocumentError {
   error: TextAnalyticsError;
 }
 
+/**
+ * An interface representing TextAnalyticsWarning.
+ */
+export interface TextAnalyticsWarning {
+  /**
+   * Error code. Possible values include: 'LongWordsInDocument', 'DocumentTruncated'
+   */
+  code: WarningCode;
+  /**
+   * Warning message.
+   */
+  message: string;
+  /**
+   * A JSON pointer reference indicating the target object.
```

Comment:
is this like an XPath expression?

Reply:
Yeah, it's similar. JSON pointers are a specific representation for selecting nodes from an object. We could maybe rehydrate these references in the client, but that will have some unfortunate consequences like not being able to serialize the data if one of these JSON pointers was returned in the response.